Formula/todoman.rb

```diff
@@ -0,0 +1,79 @@
+class Todoman < Formula
+  include Language::Python::Virtualenv
+
+  desc "Simple CalDAV-based todo manager"
+  homepage "https://todoman.readthedocs.io/"
+  url "https://pypi.python.org/packages/2f/06/a377b38dcd3cba5334f3e6444ab5a26a0d7b134a6d040ac46a16ed70568f/todoman-1.8.0.tar.gz"
+  sha256 "8ccd602da8b0870dab8273dd50631847b2b9ad7e902a8ec5bfe901e0ea6c72a2"
```

Comment:
The GitHub tarballs don't work, because they lack git metadata.  
The tarballs at PyPI have a python file which is generated when publishing, which contains the required metadata.